src/posix/platform/netif.cpp

```diff
@@ -210,6 +210,11 @@ static uint8_t            sAddedExternalRoutesNum = 0;
 static otIp6Prefix        sAddedExternalRoutes[kMaxExternalRoutesNum];
 #endif
 
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_ENABLE && OPENTHREAD_CONFIG_NAT64_TRANSLATOR_ENABLE
+static constexpr uint32_t kNat64RoutePriority =
+    100; ///< Priority for the route to NAT64 CIDR, 100 means a high priority route.
```

Comment:
nit: to fit on one line
```suggestion
static constexpr uint32_t kNat64RoutePriority = 100; ///< Priority for route to NAT64 CIDR, 100 means a high priority.
```